includes/models/model.llms.student.php

```diff
@@ -1253,6 +1285,42 @@ private function insert_incompletion_postmeta( $object_id, $trigger = 'unspecifi
 
 	}
 
+	/**
+	 * Add student postmeta data when lesson is favorited.
+	 *
+	 * @since [version]
+	 *
+	 * @see LLMS_Student->mark_favorite()
+	 *
+	 * @param int $object_id WP User ID.
+	 * @return boolean
+	 */
+	private function insert_favorite_postmeta( $object_id ) {
+
+		$update = llms_update_user_postmeta( $this->get_id(), $object_id, '_favorite', true );
+
+		// Returns an array with errored keys or true on success.
```

Comment:
doesn't return an array: again this is a copy&paste thing, this comment is wrong from the source but why carrying something wrong on? :D

Reply:
Fixed. My fault, I didn't notice, just focused on getting it done 😅